using_images/other_images/jenkins.adoc

```diff
@@ -606,67 +618,55 @@ documentation].
 
 For more details on the sample job included in this image, see this link:https://github.com/openshift/origin/blob/master/examples/jenkins/README.md[tutorial].
 
-[[pipeline-plug-in]]
-== {product-title} Pipeline Plug-in
-
-The Jenkins image's list of pre-installed plug-ins includes the OpenShift
-Pipeline plug-in, which assists in the creation of CI/CD workflows in Jenkins
-that run against an {product-title} server. A series of build steps, post-build
-actions, and SCM-style polling are provided, which equate to administrative and
-operational actions on the {product-title} server and the API artifacts hosted
-there.
-
-In addition to being accessible from the classic "freestyle" form of Jenkins
-job, the build steps as of version 1.0.14 of the {product-title} Pipeline
-Plug-in are also available to Jenkins Pipeline jobs via the DSL extension points
-provided by the Jenkins Pipeline Plug-in. The link:https://github.com/openshift/origin/tree/master/examples/jenkins/pipeline[OpenShift Jenkins Pipeline build strategy sample]
-illustrates how to use the OpenShift Pipeline plugin DSL versions of its steps.
-
-The
-https://github.com/openshift/jenkins/tree/master/1/contrib/openshift/configuration/jobs/OpenShift%20Sample[sample
-Jenkins job] that is pre-configured in the Jenkins image utilizes the
-{product-title} pipeline plug-in and serves as an example of how to leverage the
-plug-in for creating CI/CD flows for {product-title} in Jenkins.
-
-See the link:https://github.com/openshift/jenkins-plugin/[the plug-in's README] for a
-detailed description of what is available.
-
-[[client-plugin-in]]
-== {product-title} Client Plug-in
-
-The experiences gained working with users of the OpenShift Pipeline plug-in,
-coupled with the rapid evolution of both Jenkins and OpenShift, have provided
-valuable insight into how to integrate {product-title} from Jenkins jobs.
-
-As such, the new experimental
-link:https://github.com/openshift/jenkins-client-plugin[OpenShift Client Plug-in
-for Jenkins] is now offered as a technical preview and is included in the
-OpenShift Jenkins images on CentOS
-(*docker.io/openshift/jenkins-1-centos7:latest* and
-*docker.io/openshift/jenkins-2-centos7:latest*). The plug-in is also available
-from the Jenkins Update Center. The OpenShift Client plug-in will eventually
-replace the OpenShift Pipeline plug-in as the tool for OpenShift integration
-from Jenkins jobs. The OpenShift Client Plug-in provides:
+[[plug-ins]]
+== Jenkins Plug-ins
+
+The following plug-ins are provided to integrate Jenkins with {product-title}.
+They are available by default in the Jenkins image.
+
+[[jenkins-pipeline-dsl-plug-in]]
+=== {product-title} Jenkins Pipeline DSL Plug-in
```

Comment:
we tend to call this the Jenkins OpenShift Client Plug-in when discussing it, i'd prefer to call it that here. (though I see the github repo doesn't reflect that).  But it is what it's indexed as:
https://wiki.jenkins.io/display/JENKINS/OpenShift+Client+Plugin

So we should just fix the readme in the repo.  (maybe @gabemontero can take care of that).